packages/runtime/src/extensibility/facades/anonymous/VersionFacade.ts

```diff
@@ -0,0 +1,11 @@
+import { Result } from "@js-soft/ts-utils";
+import { Inject } from "typescript-ioc";
+import { CheckBackboneCompatibilityUseCase } from "../../../useCases/anonymous/CheckBackboneCompatibilityUseCase";
+
+export class VersionFacade {
```

Comment:
Version -> BackboneVersion or BackboneCompatibility

same for everywhere in this PR. I prefer BackboneCompatibility. The only place where the wording is valid is the VersionClient in transport.